frontend/src/pages/modelServing/useServingPlatformStatuses.ts

```diff
@@ -11,7 +11,13 @@ const useServingPlatformStatuses = (): ServingPlatformStatuses => {
   const kServeInstalled = !!kServeStatus.requiredComponents?.[StackComponent.K_SERVE];
   const modelMeshInstalled = !!modelMeshStatus.requiredComponents?.[StackComponent.MODEL_MESH];
 
-  const { isNIMAvailable } = React.useContext(NIMAvailabilityContext);
+  const { isNIMAvailable, refresh } = React.useContext(NIMAvailabilityContext);
+  React.useEffect(() => {
+    refresh().catch((error) =>
+      /* eslint-disable-next-line no-console */
+      console.error('Failed to refresh NIM availability:', error),
+    );
+  });
```

Comment:
```suggestion
  // eslint-disable-next-line react-hooks/exhaustive-deps
  }, []);
```

Not have `,[]` will cause an infinite loop probably. the `// eslinst-disable` will allow help you run it once on load 


(we have lint rules that want it all to run, technically you would have a `const [runOnce, setRunOnce] = React.useState(false)` and in the react effect set the run once and check if it's already run before running it, but that's verbose)